package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "werewolf-brain",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "description": "Werewolf game brain",
-  "main": "./dist/index.js",
+  "main": "index.js",
   "scripts": {
     "build": "webpack",
```

Comment:
This should also be removed

Reply:
Yeah, Travis is failing because of that.